Configuration/DataProcessing/python/Impl/AlCaHcalIsoTrk_Run3.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+"""
+_AlCaHcalIsoTrk_Run3_
+
+Scenario supporting proton collisions for AlCa needs for AlCaHcalIsoTrk data stream
+
+"""
+
+from Configuration.DataProcessing.Scenario import *
+from Configuration.DataProcessing.Utils import stepALCAPRODUCER,dictIO,gtNameAndConnect
+from Configuration.Eras.Era_Run3_cff import Run3
+from Configuration.DataProcessing.Impl.pp import pp
+import FWCore.ParameterSet.Config as cms
+
+class AlCaHcalIsoTrk_Run3(pp):
```

Comment:
@gpetruc the other Alca are deriving from the Alca implementation `class AlCaLumiPixels_Run3(AlCa):` instead of pp ?